code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -755,3 +755,9 @@
 	icon_state = "lean"
 	list_reagents = list(/datum/reagent/consumable/lean = 20)
 	random_sprite = FALSE
+
+/obj/item/reagent_containers/food/drinks/colocup/lean/suicide_act(mob/living/carbon/user)
+	user.say("I LOVE LEAN!!")
+	user.visible_message(span_suicide("[user] begins to chug the cup of lean and is starting to turn purple! It looks like [user.p_theyre()] trying to commit suicide!"))
```

Comment:
Fuckin hell shoulda removed this when I had the chance, Anyway

visible message takes quite a few optional inputs, in this case I think `self_message` could be useful

Reply:
i dont see how I could use that here
every other suicide act uses visible_message

Reply:
self_message lets you show something different to the user then you do to everyone else. it's not a new proc, it's just a second argument to it

Reply:
Most suicides dont have that
![image](https://user-images.githubusercontent.com/89947174/158087737-94a643f1-49e6-441b-b626-dadc34e9f013.png)
![image](https://user-images.githubusercontent.com/89947174/158087744-cc0d4ae1-75da-431a-8452-be795dd60c3d.png)
![image](https://user-images.githubusercontent.com/89947174/158087754-e88c58d6-6fed-436d-9978-9cee176af07f.png)
![image](https://user-images.githubusercontent.com/89947174/158087762-586a9caa-76e0-4bbb-937d-dbd3412b1c1e.png)

But I will add it